com.github.emmanueltouzery.projectpad.json

```diff
@@ -0,0 +1,99 @@
+{
+  "app-id": "com.github.emmanueltouzery.projectpad",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
```

Comment:
```suggestion
  "runtime-version": "20.08",
```

Reply:
thank you! i otherwise already have another flatpak and wanted to keep that runtime up-to-date but couldn't find any info about the latest version. In https://docs.flatpak.org/en/latest/available-runtimes.html they talk about 19.08 and I couldn't find a way to list available runtimes in the flatpak CLI? Otherwise I tested locally and things are OK also with that newer runtime.